controllers/sitemap.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+const express = require('express');
+const sitemap = require('sitemap');
+const compose = require('lodash/fp/compose');
+const concat = require('lodash/fp/concat');
+const sortBy = require('lodash/fp/sortBy');
+const uniqBy = require('lodash/fp/uniqBy');
+
+const contentApi = require('../common/content-api');
+const { getBaseUrl } = require('../common/urls');
+const { sMaxAge } = require('../middleware/cached');
+
+const router = express.Router();
+
+/**
+ * Build a flat list of all canonical routes
+ * Combines application routes and routes defined by the CMS
+ */
+async function getCanonicalRoutes() {
+    /**
+     * Static routes
+     *
+     * Used for pages that are outside of the CMS, either:
+     * - Hard-coded pages (like homepage and 10k pages)
+     * - Landing pages (where child pages are powered by the CMS, but landing is custom)
+     */
+    const staticRoutes = [
```

Comment:
This list was previously generated from `routes.js`. The biggest problem with this is that it only generated entries for the _mount-path_ of a section, i.e. didn't account for sub-pages like `/funding/programmes/all`.

Given that only a handful of pages live outside the CMS this feels simpler to hard-code.

Reply:
👍 and a great motivator (and helpful list) to get most of the remaining stuff into the CMS, too.